### DISCUSSION TITLE
Transfer-Encoding: chunked erroe

### DISCUSSION THREAD

**Discussion by peterzyk** [projectdiscovery/nuclei#4139] [Q&A]
Inexplicably added to the data packet, chunked transmission

POST /servlet/~ic/bsh.servlet.BshServlet HTTP/1.1
Host: 18.136.224.89:8080
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2762.73 Safari/537.36
**Transfer-Encoding: chunked**
Accept: */*
Accept-Encoding: gzip, deflate
Accept-Language: en
Connection: close
Content-Type: application/x-www-form-urlencoded

bsh.script=pwd()
[chunked error.md](https://github.com/projectdiscovery/nuclei/files/12571430/chunked.error.md)

**Comment by ehsandeep** [comment]
@peterzyk It's known issue and will be fixed in the next release, GH issue - https://github.com/projectdiscovery/nuclei/issues/4032

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
